src/utils/taskOverDueDiscordMembers.ts

```diff
@@ -20,6 +20,7 @@ export const taskOverDueDiscordMembers = async (): Promise<
 
     return assigneeIds;
   } catch (e) {
-    return errors.INTERNAL_SERVER_ERROR;
+    console.log(e);
+    throw e;
```

Comment:
NIT - why console.log and why not console. error?
in my opining console.error will be much suited option here